package.json

```diff
@@ -200,6 +203,7 @@
 		"postcss-loader": "^3.0.0",
 		"postcss-preset-env": "^6.7.0",
 		"prettier": "npm:wp-prettier@^2.2.1-beta-1",
+		"puppeteer": "^10.1.0",
```

Comment:
Is this needed since we already have puppeteer installed? 🤔 This may be why there are problems in E2E as well.

Reply:
@aaemnnosttv Nice!  I was trying to work this out for ages!  :joy: 

That explains why everything was failing for me locally even on develop :joy:   I should have nuked `node_modules` and tried again.

All working now :+1: